qa/L0_trace/test.sh

```diff
@@ -80,8 +80,77 @@ cp -r $DATADIR/$MODELBASE $MODELSDIR/simple && \
 
 RET=0
 
+# Helpers =======================================
+function assert_curl_success {
+  message="${1}"
+  if [ "$code" != "200" ]; then
+    cat ./curl.out
+    echo -e "\n***\n*** ${message} : line ${BASH_LINENO}\n***"
```

Comment:
I think these `BASH_LINENO` will statically call the line number of these functions (ex: 88 or 97), rather than the line of the caller. And I believe it's unexpected complicated to correctly pass line numbers around in bash. So I'd probably just take these out.

Reply:
When I was making adjustments to the script I tested the error messages (accidentally by making mistakes in the logic) and they worked as expected. `BASH_LINENO` should not statically call the line number it's invoked on, I believe `LINENO` would do that. Instead, according to the documentation `BASH_LINENO` should evaluate to the line number its parent function was called on, which is what I observed. If we still prefer to remove `BASH_LINENO` altogether, I can incorporate the above suggestions and pass additional specification parameters.

Reply:
> BASH_LINENO should evaluate to the line number its parent function was called on, which is what I observed. 
Oh nice, TIL. LGTM locally too. Thanks for pushing back :slightly_smiling_face: